charts/argocd-apps/ci/applications-values.yaml

```
@@ -20,6 +20,7 @@ applications:
     automated:
       prune: false
       selfHeal: false
+  revisionHistoryLimit: 5
```

Maybe `null` or `""` would be a better default:

```
kubectl explain applications.spec
KIND:     Application
VERSION:  argoproj.io/v1alpha1

RESOURCE: spec <Object>

DESCRIPTION:
     ApplicationSpec represents desired application state. Contains link to
     repository with application definition and additional parameters link
     definition revision.

FIELDS:
   (..)

   revisionHistoryLimit <integer>
     RevisionHistoryLimit limits the number of items kept in the application's
     revision history, which is used for informational purposes as well as for
     rollbacks to previous versions. This should only be changed in exceptional
     circumstances. Setting to zero will store no history. This will reduce
     storage used. Increasing will increase the space used to store the history,
     so we do not recommend increasing it. Default is 10.

   (..)
```

> so **we do not recommend increasing it. Default is 10**.

---

```suggestion
  revisionHistoryLimit: null
```
Is this a valid value for revisionHistoryLimit if \<integer\> is expected?

---

`null` into `with .revisionHistoryLimit` would not render that block

---

Great, thanks! Committing the change